lib/authenticators/ldap/authn.rb

```diff
@@ -0,0 +1,40 @@
+require 'active_support'
```

Comment:
This file looks like a duplicate of a class in `lib/authenticators/ldap/authn_ldap.rb`.  Do we need them both?